data_safe_haven/infrastructure/stacks/shm/firewall.py

```diff
@@ -463,12 +354,18 @@ def __init__(
                             ],
                             source_addresses=[props.subnet_update_servers_iprange],
                             target_fqdns=[
+                                "apt.postgresql.org",
                                 "archive.ubuntu.com",
                                 "azure.archive.ubuntu.com",
                                 "changelogs.ubuntu.com",
                                 "cloudapp.azure.com",  # this is where azure.archive.ubuntu.com is hosted
+                                "d20rj4el6vkp4c.cloudfront.net",
+                                "dbeaver.io",
+                                "packages.gitlab.com",
                                 "packages.microsoft.com",
+                                "qgis.org",
                                 "security.ubuntu.com",
+                                "ubuntu.qgis.org"
```

Comment:
We're not currently using these additional repository sources. Can you leave them here (in case we want to use them again) but comment them out?